dds/DCPS/DataReaderImpl.cpp

```diff
@@ -1221,10 +1221,8 @@ DataReaderImpl::enable()
         " Cached_Allocator_With_Overflow %x with %d chunks\n",
         rd_allocator_.get(), n_chunks_));
 
-  if ((qos_.liveliness.lease_duration.sec !=
-      DDS::DURATION_INFINITE_SEC) &&
-      (qos_.liveliness.lease_duration.nanosec !=
-          DDS::DURATION_INFINITE_NSEC)) {
+  if ((qos_.liveliness.lease_duration.sec != DDS::DURATION_INFINITE_SEC) &&
+      (qos_.liveliness.lease_duration.nanosec != DDS::DURATION_INFINITE_NSEC)) {
```

Comment:
Why are these lines changed in this PR? Doesn't seem related to this task.

Reply:
for easier to read when I was combing through the code for potential relevant changes.

Reply:
I don't think this needs to change here.  We should try to minimize churn so that someone looking at `git blame` can see when the code actually changes.